lib/Readline.pm

```diff
@@ -693,8 +693,14 @@ class Readline:ver<0.1.5> {
         tgetnum('');
       }
       when 'darwin' {
+        # needs homebrew installed version of readline
+        # macOS ships with incompatible libedit
         $library-match = rx/:i libreadline\.(\d+)\.dylib $/;
-      }
+        # set version to v0.0 so we can check if it was found
+        $version = v0.0;
+        # homebrew directory
+        @library-path = ('/usr/local/opt/readline/lib')
```

Comment:
you'd need https://github.com/lathropd/perl6-readline/pull/2/files to support Apple silicon